store/rootmulti/store.go

```diff
@@ -64,9 +63,6 @@ type Store struct {
 	interBlockCache  types.MultiStorePersistentCache
 	iavlCacheManager types.CacheManager
```

Comment:
This cachemanager still remains. Do you wanted to remain it? Or did you miss it?

Reply:
The removal of iavlCacheManager is progressed in pr 595.

Reply:
This PR is for a fix to remove Prefetch functionality and the fix to remove `CacheManager` is in progress by #595.
The above fix seems to be [here](https://github.com/line/lbm-sdk/pull/595/commits/a583ff613afc6a4746a39fd0837e230ebdeae830#diff-2bc7a86169d4c2f891d7ae74b21ca4bcc5787d10114e2eee215fe1ddba5c999eL64-L65).